API/Program.cs

```diff
@@ -10,16 +9,8 @@ public static void Main(string[] args)
         {
             var builder = WebApplication.CreateBuilder(args);
 
-        
-            // Add services to the container.
+            builder.Services.AddServices(builder.Configuration);
```

Comment:
And now you have the same situation, but now your program have only one line of services. We must see what services we register in our di, but if you have additional configs, you have to create extension method